public/javascripts/SVLabel/src/SVLabel/Keyboard.js

```diff
@@ -54,6 +97,24 @@ function Keyboard ($) {
      * @private
      */
     function documentKeyUp (e) {
+        switch (e.keyCode) {
+            // "left"
+            case 37:
+                movePano(270);
```

Comment:
Should left and right keys make people step on sides or change the heading angle? The default GSV API changes the heading angle. What is your rational for stepping left or right?